tests/test_h5store.py

```diff
@@ -148,215 +150,215 @@ class H5StoreTest(BaseH5StoreTest):
+    def test_delete(self,setUp):
         with self.open_h5store() as h5s:
             key = 'delete'
             d = self.get_testdata()
             h5s[key] = d
-            self.assertEqual(len(h5s), 1)
-            self.assertEqual(h5s[key], d)
+            assert len(h5s) == 1
+            self.assertEqual( h5s[key] , d)
             del h5s[key]
-            self.assertEqual(len(h5s), 0)
-            with self.assertRaises(KeyError):
+            assert len(h5s) == 0
+            with pytest.raises(KeyError):
                 h5s[key]
 
-    def test_update(self):
+    def test_update(self,setUp):
         with self.open_h5store() as h5s:
             key = 'update'
             d = {key: self.get_testdata()}
             h5s.update(d)
-            self.assertEqual(len(h5s), 1)
+            assert len(h5s) == 1
             self.assertEqual(h5s[key], d[key])
 
-    def test_clear(self):
+    def test_clear(self,setUp):
         with self.open_h5store() as h5s:
             h5s.clear()
             key = 'clear'
             d = self.get_testdata()
             h5s[key] = d
-            self.assertEqual(len(h5s), 1)
-            self.assertEqual(h5s[key], d)
+            assert len(h5s) == 1
+            self.assertEqual( h5s[key] , d)
             h5s.clear()
-            self.assertEqual(len(h5s), 0)
+            assert len(h5s) == 0
 
-    def test_reopen(self):
+    def test_reopen(self,setUp):
         with self.open_h5store() as h5s:
             key = 'reopen'
             d = self.get_testdata()
             h5s[key] = d
         with self.open_h5store() as h5s:
-            self.assertEqual(len(h5s), 1)
-            self.assertEqual(h5s[key], d)
+            assert len(h5s) == 1
+            self.assertEqual( h5s[key] , d)
 
-    def test_open_twice(self):
+    def test_open_twice(self,setUp):
         h5s = self.get_h5store()
         h5s.open()
         try:
-            with self.assertRaises(H5StoreAlreadyOpenError):
+            with pytest.raises(H5StoreAlreadyOpenError):
                 h5s.open()
         finally:
             h5s.close()
 
-    def test_open_reentry(self):
+    def test_open_reentry(self,setUp):
         with self.open_h5store() as h5s:
             with h5s:
                 pass
 
-    def test_reopen_explicit_open_close(self):
+    def test_reopen_explicit_open_close(self,setUp):
         h5s = self.get_h5store().open()
         key = 'reopen'
         d = self.get_testdata()
         h5s[key] = d
         h5s.close()
         h5s.open()
-        self.assertEqual(len(h5s), 1)
-        self.assertEqual(h5s[key], d)
+        assert len(h5s) == 1
+        self.assertEqual( h5s[key] , d)
         h5s.close()
 
-    def test_write_valid_types(self):
+    def test_write_valid_types(self,setUp):
         with self.open_h5store() as h5s:
             for k, v in self.valid_types.items():
                 h5s[k] = v
-                self.assertEqual(h5s[k], v)
+                self.assertEqual( h5s[k] , v)
```

Comment:
```suggestion
                self.assertEqual(h5s[k], v)
```
Please also revert everywhere else where there is the same formatting issue.